server/scripts/update-operators.sh

```diff
@@ -11,14 +12,18 @@ if [ ! -d "data/community-operators" ]; then
   fi
 
   git clone $OPERATORS_REPO community-operators || exit 1
-  popd
+fi
 
-  exit 0
+if [ ! $2 ]; then
+  echo "Using master branch"
+  OPERATORS_BRANCH=master
+else
+  OPERATORS_BRANCH=$2
 fi
 
-pushd data/community-operators
-git fetch origin master
-git pull origin master
+pushd community-operators
+git fetch origin $OPERATORS_BRANCH
+git pull origin $OPERATORS_BRANCH
```

Comment:
Should this be the only change? Current pushd/popd changes leave one level down (in ./data).